### PR TITLE
Option to stop following canonical url

### PR DESCRIPTION
**Pull request by JanPetterMG**
Alternative fix for #232 

An simple on/off switch.
Either you follow/redirect to the canonical url (when discovered), or you simply doesn't.

Simply put `follow_canonical` (`bool`) in your config array. Default value: `true`

### CODE REVIEW

**Comment by JanPetterMG** [comment]
You'll now have the ability to choose whatever conditions you want whether to follow/redirect or not. If you decide to follow the detected url, simply re-run Embed using the canonical url instead.

File: src/Adapters/Adapter.php [hunk 1]
```diff
@@ -299,6 +299,11 @@ public function getHtml()
     public function getUrl()
     {
         $default = (string) $this->getResponse()->getUrl();
+
+        if ($this->getConfig('follow_canonical') === false) {
+            return $default;
+        }
+
```
Comment (oscarotero) [inline src/Adapters/Adapter.php]: The option cannot be evaluated here. This function just return the canonical url, doesn't follow it. It must be checked here: https://github.com/oscarotero/Embed/blob/master/src/Embed.php#L73

Comment (JanPetterMG) [inline src/Adapters/Adapter.php]: Sorry, that was unintended. It's fixed now.

**Comment by oscarotero** [comment]
👍 thanks

**Pull request merged**